install/source/include-get-the-source.rst

```diff
@@ -5,6 +5,6 @@ or find the initial version at https://ftp.zammad.com.
 
    $ cd /opt
    $ wget https://github.com/zammad/zammad/archive/stable.tar.gz
-   $ tar -xzf stable.tar.gz -C zammad
+   $ tar -xzf stable.tar.gz --strip-components 1 -C zammad
```

Comment:
When I do this on my mac, it fails because the `zammad` directory is missing. Is this due to an outdated `tar`? Or should we include an `mkdir`?

```
❯ tar --version
bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6
> tar -xzf stable.tar.gz --strip-components 1 -C zammad
tar: could not chdir to 'zammad'
```

Reply:
That's not required because we're creating the user before:
https://github.com/zammad/zammad-documentation/blame/a03820859b97b430434bfe14190c901349e50e94/install/source.rst#L45

This automatically adds the required dir.
If you skip that step you'd have to manually add the directory-  however, you'd also run Zammad with a privileged user or in unsupported scope.